cumulus/pallets/xcmp-queue/src/migration.rs

```diff
@@ -25,14 +27,14 @@ use frame_support::{
 };
 
 /// The current storage version.
-pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(3);
+pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(4);
 
 pub const LOG: &str = "runtime::xcmp-queue-migration";
 
 /// Migrates the pallet storage to the most recent version.
-pub struct MigrationToV3<T: Config>(PhantomData<T>);
+pub struct MigrationToV4<T: Config>(PhantomData<T>);
```

Comment:
Could you split up all these migrations into individual `MigrateToVX` structs and then also use the `VersionedMigration` from `frame_support` please.

Reply:
I didn't know about `VersionedMigration`. Done. PTAL.